charts/zalenium/values.yaml

```diff
@@ -183,15 +183,15 @@ persistence:
     folder: ""
 
 ingress:
-  enabled: false
-  tls: false
+  enabled: true
+  tls: true
   # secretName: my-tls-cert # only needed if tls above is true
-  hostname: zalenium.foobar.com
+  hostname: zalenium.qa-aks.qa.nexploredevops.com
```

Comment:
In general the default values shouldn't be modified, but set when installing the chart. In this particular case, you tie the values for a particular environment, so you're not actually allowing for a general install without customization.
Same for tls. Seems good to enable it now that you're on a real cluster with managed certificates, but for local tests with minikube you're forcing to set values on install time.